### PR TITLE
Build installer image in azure integration

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- currently, the azure jobs reuse the latest published installer image. the new scripts try to build the installer when used in presubmit hooks

**Related issue(s)**

### CODE REVIEW

File: prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh [hunk 1]
```diff
@@ -169,45 +190,48 @@ fi
...
-    -o installer-cr-gardener-azure.yaml.tpl \
-    -o installer-config-production.yaml.tpl \
-    -o installer-config-azure-eventhubs.yaml.tpl \
-    --timeout 90m
+    set -x
+    INSTALLATION_RESOURCES_DIR=${KYMA_PROJECT_DIR}/installation/resources
+    kyma install \
+        --ci \
+        --source $KYMA_INSTALLER_IMAGE \
+        -o "${INSTALLATION_RESOURCES_DIR}"/installer-cr-azure-eventhubs.yaml.tpl \
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh:231]: 👍

File: prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh [hunk 2]
```diff
@@ -102,6 +117,12 @@ cleanup() {
         az group delete -n "${RS_GROUP}" -y
     fi
 
+    if [ -n "${CLEANUP_DOCKER_IMAGE}" ]; then
+        shout "Delete temporary Kyma-Installer Docker image"
+        date
+        "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/delete-image.sh"
+    fi
+
     rm -rf "${TMP_DIR}"
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh:126]: I see `TMP_DIR=$(mktemp -d)` was added, however, `rm -rf "${TMP_DIR}"` was already there. So I am wondering if TMP_DIR was already set before 🤔 If yes, then we are not deleting the previous TMP_DIR anymore.

Comment (k15r) [reply prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh:126]: I could not find any trace of a previous tmp_dir, lets see what happens once we execute the script... local testing is a bit limited for those scripts

Comment (k15r) [reply prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh:126]: on the other hand... this is meant to be executed in a docker container so the cleanup of folders is kinda useless, don't you think?

**Pull request merged**